package.json

```diff
@@ -22,9 +22,10 @@
     "test:cucumber-super-admin": "cucumber-js --retry 5 --parallel 8 --require-module 'ts-node/register' --require 'src/test/functional/**/*.ts' --tags 'not @skipped and not @pending' src/test/functional/features/super_admin/*.feature",
     "test:cucumber-admin": "cucumber-js --retry 3 --parallel 3 --require-module 'ts-node/register' --require 'src/test/functional/**/*.ts' --tags 'not @skipped and not @pending' src/test/functional/features/admin/*.feature",
     "test:functional1": "yarn test:cucumber-viewer && yarn test:cucumber-super-admin && yarn test:cucumber-admin && yarn test:pa11y",
-    "test:functional": "npm-run-all -scl test:codecept test:functional:report test:pa11y",
+    "test:functional": "npm-run-all -scl test:codecept test:codecept:report test:pa11y",
```

Comment:
can you rename the test:pa11y to test:a11y and move the config over. Also remove the test:pa11y step from functional and put it in cichecks which is where it should be done, if it fails may need to move back as was done for a reason. 

(cichecks should use the new test:a11y